proto/ibc/applications/fee/v1/tx.proto

```diff
@@ -104,7 +104,7 @@ message MsgPayPacketFeeResponse {}
 // MsgPayPacketFeeAsync defines the request type for the PayPacketFeeAsync rpc
 // This Msg can be used to pay for a packet at a specified sequence (instead of the next sequence send)
 message MsgPayPacketFeeAsync {
-  option (cosmos.msg.v1.signer)      = "packet_fee.refundaddress";
+  option (cosmos.msg.v1.signer)      = "packet_fee";
```

Comment:
We should have more docs for `GetSigners` annotation, as it may not be clear for everyone how to set them for inner messages:

cc https://github.com/cosmos/cosmos-sdk/issues/16294

Reply:
Fixed on main